src/widgets/AfterInteractions.js

```diff
@@ -1,20 +1,27 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import { ActivityIndicator, InteractionManager } from 'react-native';
+import { useIsFocused } from '@react-navigation/native';
 
 import { FlexView } from './FlexView';
 import { SUSSOL_ORANGE } from '../globalStyles/index';
 
 export const AfterInteractions = ({ children, placeholder }) => {
   const [ready, setReady] = useState(false);
 
+  const focused = useIsFocused();
+
+  useEffect(() => {
+    setReady(focused);
+  }, [focused]);
+
   useEffect(() => {
     InteractionManager.runAfterInteractions(() => {
-      setReady(true);
+      setReady(focused);
     });
-  }, []);
+  }, [focused]);
 
-  return ready ? (
+  return focused && ready ? (
```

Comment:
With this, this works when you go back rather than when you push onto the navigation stack